src/Pester.Runtime.ps1

```diff
@@ -680,7 +680,7 @@ function Invoke-TestItem {
 
                 $Test.FrameworkData.Runtime.ExecutionStep = 'Finished'
 
-                if ($Result.ErrorRecord.FullyQualifiedErrorId -eq 'PesterTestSkipped') {
+                if (@('PesterTestSkipped', 'PesterTestInconclusive', 'PesterTestPending') -contains $Result.ErrorRecord.FullyQualifiedErrorId) {
```

Comment:
@fflaten  what you commented on in the Issue happens here. The error record id is translated to .Skipped = $true on the test object. So all the accounting that is done above this is correct, but you can still see the actual reason for marking the test as skipped on the ErrorRecord itself on the Result object.

Reply:
Thanks 🙂  Didn't remember that `Add-RSpecTestObjectProperties` would set the correct result based on `Skipped = $true` during teardown.